src/api/index.js

```diff
@@ -1,5 +1,7 @@
 export { executeSQL } from './SQL';
 export { getUserDatasets } from './UserDatasets';
-export {
+/* export {
```

Comment:
A confirmation with Product is required to further talk about removing the sql pure filters..., once this has been merged 
cc @borja-munoz